Fix Javascript lint issues

Etheryte opened the pull request:
## What does this PR change?

Fixes lint issues unrelated to the the Flow problem.

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: No gui diff.

- [x] **DONE**

## Test coverage
- No tests: Only linting fixes.

- [x] **DONE**

## Links

- [x] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)


## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

mcalmer commented:
@Etheryte I removed me from the reviewer list. I have no clue about javascript. It makes no sense to get a review from me :-)

Etheryte commented:
> @Etheryte I removed me from the reviewer list. I have no clue about javascript. It makes no sense to get a review from me :-)

That's fair enough, I added you to the list since you brought the issue to my attention.

ncounter reviewed (approved):
Good leftover catches! :+1:
